pkg/systemd/generate/pods.go

```diff
@@ -252,6 +252,7 @@ func generatePodInfo(pod *libpod.Pod, options entities.GenerateSystemdOptions) (
 		StopTimeout:       stopTimeout,
 		GenerateTimestamp: true,
 		CreateCommand:     createCommand,
+		RunRoot:           pod.Runtime().RunRoot(),
```

Comment:
This looks unrelated? Also you could just use `infraCtr`, so you do not have to add the new
`func (p *Pod) Runtime() *Runtime` function

Reply:
Yes, that's a drive-by fix.  I can repush but it feels like a half-full half-empty thing.

Reply:
The thing here you did not change any tests. It does not seem to part of the template so this is pointless, no?

Reply:
It's part of the template but not checked for in the e2e tests.  The unit tests set RunRoot.  I can add a test but did not find it worth.

Reply:
I'll add an e2e test for it :^) Better safe than sorry